src/Integration.Vsix/Analysis/IAnalyzerController.cs

```diff
@@ -30,7 +31,13 @@ public interface IAnalyzer
     {
         bool IsAnalysisSupported(IEnumerable<AnalysisLanguage> languages);
 
-        void ExecuteAnalysis(string path, string charset, IEnumerable<AnalysisLanguage> detectedLanguages, IIssueConsumer consumer, ProjectItem projectItem, CancellationToken cancellationToken);
+        void ExecuteAnalysis(string path,
+            string charset,
+            IEnumerable<AnalysisLanguage> detectedLanguages,
+            IIssueConsumer consumer,
+            ProjectItem projectItem,
+            CancellationToken cancellationToken,
+            IAnalyzerOptions analyzerOptions);
```

Comment:
The convention for APIs with a `CancellationToken` is to have it as the last parameter.